packages/gafl-webapp-service/src/pages/multibuy/remove-licence/__tests__/update-transaction.spec.js

```diff
@@ -85,13 +96,22 @@ describe('remove-licence > update transaction', () => {
     expect(mockRequest.cache().helpers.status.setCurrentPermission()).toEqual(expect.objectContaining(setPermission))
   })
 
-  it('transaction.set is being called with a permission removed', async () => {
+  it('transaction.set is being called with a permission removed with no duplciates', async () => {
```

Comment:
You need to alter this description to be a bit more 'plain English', something like 'A permission is removed on request when there are no duplicate permissions'